CalibTracker/SiStripCommon/BuildFile.xml

```diff
@@ -4,7 +4,6 @@
 <use name="DataFormats/SiStripDetId"/>
 <use name="DataFormats/Scalers"/>
 <use name="DataFormats/OnlineMetaData"/>
-<use name="SimDataFormats/TrackingAnalysis"/>
```

Comment:
@guitargeek that's gonna conflict with https://github.com/cms-sw/cmssw/pull/31643. Hopefully after staying idle for 15 days will be merged...

Reply:
Thanks for letting me know! I'll get the CalibTracker changes out of this PR then.